northernco/NorthernMaintenanceModeBundle
# Maintenance Mode Bundle

### Task
N/A

### Goal
- Create maintenance mode bundle.

### Testing Instructions
- Add to project, run `app:maintenance --enable` to enable, `app:maintenance --disable to disable`
- Ensure that README docs make sense
- Ensure that you can override the template by creating a `templates/bundles/NorthernMaintenanceModeBundle/maintenance.html.twig` file.

### Reviewers
@natewiebe13
